src/Microsoft.DocAsCode.Dfm/Rules/DfmVideoBlockRule.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm
+{
+    using System.Text.RegularExpressions;
+
+    using Microsoft.DocAsCode.MarkdownLite;
+
+    public class DfmVideoBlockRule : IMarkdownRule
+    {
+        private static readonly Regex _videoRegex = new Regex(@"^ *\[\!Video +(?<link>https?\:\/\/.+?) *\] *(\n|$)", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
The syntax you mentioned has already been supported in our syntax.
However, That's a picture link of video syntax but not embeded video in the page. They are different.

---
In reply to: [72000890](https://github.com/dotnet/docfx/pull/526#discussion_r72000890) [](ancestors = 72000890)